src/AppShed/Extensions/SpreadsheetBundle/Controller/ReadController.php

```diff
@@ -137,7 +141,27 @@ public function documentAction(Request $request)
                             $link->setScreenLink($innerScreen);
                         } else {
                             if (!empty($value)) {
-                                $innerScreen->addChild(new HTML($value));
+
+                                $address = $doc->getAddress();
+
+                                if (!empty($address) && $name == $address ) {
```

Comment:
I think you can move the check to be inside the getAddress getter, and return null if its empty, this will protect against columns with empty names